common/src/main/java/org/cloudfoundry/autosleep/access/cloudfoundry/CloudFoundryApi.java

```diff
@@ -54,26 +36,38 @@
 import org.cloudfoundry.client.v2.events.ListEventsResponse;
 import org.cloudfoundry.client.v2.organizations.GetOrganizationRequest;
 import org.cloudfoundry.client.v2.organizations.GetOrganizationResponse;
-import org.cloudfoundry.client.v2.routes.GetRouteRequest;
-import org.cloudfoundry.client.v2.routes.GetRouteResponse;
-import org.cloudfoundry.client.v2.routes.ListRouteApplicationsRequest;
-import org.cloudfoundry.client.v2.routes.ListRouteApplicationsResponse;
-import org.cloudfoundry.client.v2.routes.RouteEntity;
+import org.cloudfoundry.client.v2.privatedomains.GetPrivateDomainRequest;
+import org.cloudfoundry.client.v2.privatedomains.GetPrivateDomainResponse;
```

Comment:
fixed in https://github.com/cloudfoundry-community/autosleep/pull/279/commits/7c1bf4bf1e8ac1ad93bd65b5648ab010ae3f44e3